elixir/lib/homework/users/user.ex

```diff
@@ -11,10 +13,37 @@ defmodule Homework.Users.User do
     timestamps()
   end
 
-  @doc false
   def changeset(user, attrs) do
     user
     |> cast(attrs, [:first_name, :last_name, :dob])
     |> validate_required([:first_name, :last_name, :dob])
+    |> validate_date(:dob)
+    |> validate_age(:dob)
+    |> unique_constraint([:first_name, :last_name, :dob], name: :users_info_unique)
```

Comment:
This isn't an ideal unique constraint.  It's possible to have two people with the same birthdate and name, and so we'd need more information such as email, home address, or other unique identifiers, but for the sake of the demonstration, there should be a constraint to keep from having duplicate people.